README.md

```diff
@@ -181,7 +181,14 @@ This is sequence diagram for communication among tapyrus-signer nodes.
 ### Round Master Flow
 
 1. Start Next Round
-     * Sleep 60 secs. (This is default value. It can be changed by --duration option.)
+     * If the round-duration timer is not started
```

Comment:
> I would prefer to use round-duration until the conclusion of discussion #143.

Yes, I agree with your preference for consistency. However, the implementation uses `interval` like `ROUND_INTERVAL_DEFAULT_SECS` since before this PR. There is no consistency since before.

I want to keep here at this time and fix consistency in another PR for #143.

Reply:
I see. 
You should remember to fix this and submit PR for #143 immediately after this PR is merged.